cmd/virtual-workspaces/options/options.go

```diff
@@ -94,6 +96,8 @@ func (o *Options) AddFlags(flags *pflag.FlagSet) {
 	o.CoreVirtualWorkspaces.AddFlags(flags)
 	o.TmcVirtualWorkspaces.AddFlags(flags)
 
+	flags.StringVar(&o.ShardExternalURL, "shard-external-url", o.ShardExternalURL, "URL used by outside clients to talk to the kcp shard this virtual workspace is related to")
```

Comment:
Are we certain that now and forever shards will be 1:1 with VW servers? Why? It's not clear to me that these need to be related, the scaling dimensions are different and one does not have storage.

Reply:
That's not exactly 1:1, but a given virtual workspace is related to one and only one shard.

@sttts Do you want to provide additional context here ?

Reply:
"is related to one and only one" ... is what I mean by `1:1` ??

Reply:
You can have a single Shard external URL for several shards (for example behind a proxy).
But you would never have a single virtual workspace URL for several shards. There is (at least) 1 per shard.
So the shard external URL is always well-defined for a given virtual workspace, though a single shard external URL
could be shared by several virtual workspaces.